pages/studentAssessmentInstanceFile/studentAssessmentInstanceFile.js

```diff
@@ -1,28 +1,16 @@
-const ERR = require('async-stacktrace');
 const express = require('express');
 const router = express.Router();
+const fileStore = require('../../lib/file-store');
 
-const config = require('../../lib/config');
-const sqldb = require('@prairielearn/prairielib/sql-db');
-const sqlLoader = require('@prairielearn/prairielib/sql-loader');
-
-const sql = sqlLoader.loadSqlEquiv(__filename);
-
-router.get('/:file_id/:display_filename', function(req, res, next) {
-    let params = {
+router.get('/:file_id/:display_filename', async function(req, res, next) {
```

Comment:
Express route handlers cannot be `async` functions. Instead we need to do this: https://github.com/PrairieLearn/PrairieLearn/blob/e78a6d01b39d88602ab7ea76088f5ecb86b6f21a/pages/workspace/workspace.js#L30

I'll fix this one.